tests/Sitecore.AspNetCore.SDK.RenderingEngine.Integration.Tests/Fixtures/TagHelpers/ImageFieldTagHelperFixture.cs

```diff
@@ -176,6 +177,72 @@ public async Task ImgTagHelper_GeneratesProperEditableImageMarkupWithCustomPrope
         sectionNode.InnerHtml.Should().Contain("src=\"/sitecore/shell/-/jssmedia/styleguide/data/media/img/sc_logo.png?mw=100&mh=50\"");
     }
 
+    [Fact]
+    public async Task ImgTagHelper_GeneratesProperSrcSetAttribute()
+    {
+        // Arrange
+        _mockClientHandler.Responses.Push(new HttpResponseMessage
+        {
+            StatusCode = HttpStatusCode.OK,
+            Content = new StringContent(Serializer.Serialize(CannedResponses.PageWithPreview))
+        });
+
+        HttpClient client = _server.CreateClient();
+
+        // Act
+        string response = await client.GetStringAsync(new Uri("/", UriKind.Relative));
+        HtmlDocument doc = new HtmlDocument();
+        doc.LoadHtml(response);
+        HtmlNode? sectionNode = doc.DocumentNode.ChildNodes.First(n => n.HasClass("component-with-images"));
+
+        // Assert
+        // Third image for <sc-img /> (index 2)
+        HtmlNode? thirdImg = sectionNode.Descendants("img").ElementAt(2);
+        thirdImg.Should().NotBeNull();
+        thirdImg.Attributes.Should().Contain(a => a.Name == "srcset");
+        thirdImg.Attributes.Should().Contain(a => a.Name == "sizes");
+
+        // Fourth image for <img /> (index 3)
+        HtmlNode? fourthImg = sectionNode.Descendants("img").ElementAt(3);
+        fourthImg.Should().NotBeNull();
+        fourthImg.Attributes.Should().Contain(a => a.Name == "srcset");
+        fourthImg.Attributes.Should().Contain(a => a.Name == "sizes");
```

Comment:
Can you update the asserts to the same style as the test below, where we're doing a more complete check on the attribute contents.